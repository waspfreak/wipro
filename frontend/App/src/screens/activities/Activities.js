import React from 'react';
import { Text, View } from 'react-native';
import styles from './styles';

export const Activities = () => {
  return (
    <View style={styles.container}>
      <Text>This is the Activity page</Text>
    </View>
  );
};
export default Activities;
