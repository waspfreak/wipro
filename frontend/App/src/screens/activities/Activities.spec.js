import React from 'react';
import Activities from './Activities';
import { render } from '@testing-library/react-native';

describe('Activities tests', () => {
  it('should match Activities snapshot', () => {
    const { baseElement } = render(<Activities />);
    expect(baseElement).toMatchSnapshot();
  });
});
