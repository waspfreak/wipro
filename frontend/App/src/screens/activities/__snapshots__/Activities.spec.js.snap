// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Activities tests should match Activities snapshot 1`] = `
<View
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={true}
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "flex": 1,
          "justifyContent": "center",
          "paddingTop": 15,
        }
      }
    >
      <Text>
        This is the Activity page
      </Text>
    </View>
  </View>
</View>
`;
